tests/src/Functional/ShowcasePageTest.php

```diff
@@ -0,0 +1,292 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_showcase\Functional;
+
+use Drupal\Tests\BrowserTestBase;
+use Drupal\node\Entity\Node;
+use Drupal\Tests\oe_showcase\Traits\AuthenticationTrait;
+use Drupal\Tests\node\Traits\ContentTypeCreationTrait;
+use Symfony\Component\DomCrawler\Crawler;
+
+/**
+ * Ensure OE Showcase Pages are created.
+ *
+ * @group oe_showcase
+ */
+class ShowcasePageTest extends BrowserTestBase {
```

Comment:
no need for the showcase as there wont be any other page tests then this
```suggestion
class PageTest extends BrowserTestBase {
```